packages/@aws-cdk/aws-servicecatalog/lib/private/association-manager.ts

```diff
@@ -33,7 +37,6 @@ export class AssociationManager {
   }
 
   public static constrainTagUpdates(portfolio: IPortfolio, product: IProduct, options: TagUpdateConstraintOptions): void {
-    this.validateCommonConstraintOptions(portfolio, product, options);
     const association = this.associateProductWithPortfolio(portfolio, product);
```

Comment:
```suggestion
    const association = this.associateProductWithPortfolio(portfolio, product, options);
```

Reply:
okay this is weird, this is my bad.  I have deja vu from string replacing these and somehow it's not in here :(